__tests__/server.js

```diff
@@ -0,0 +1,47 @@
+const db = require("../data/dbConfig");
+const request = require("supertest");
+const server = require("../api/server");
+
+afterEach(async () => {
+  await db("games").truncate();
+});
+
+describe("the route handler", () => {
+  describe("get /games", () => {
+    it("responds with 200", async () => {
+      const response = await request(server).get("/games");
+      expect(response.status).toBe(200);
+    });
+    it("responds with json", async () => {
+      const response = await request(server).get("/games");
+      expect(response.type).toMatch(/json/i);
+    });
+    it("sends the correct response object", async () => {
+      const response = await request(server).get("/games");
+      expect(response.body).toEqual([]);
+    });
+  });
```

Comment:
Clean `describe()` blocks that are hitting all the spec's laid out in the instructions, Angela 🔥